app/models/user.rb

```diff
@@ -12,7 +12,12 @@ class User < ApplicationRecord
     has_many :likes, inverse_of: :user
     #name will be treated as the displaying name in Twitter
     #I will take lastname as an optional value 
-    validates :name, :username, :email, :username, :password, presence: { message: "%{attribute} needs to have a value" }
+    validates :name, :username, :email, presence: { message: "needs to have a value" }
+  
+    # Missing password validation
+    validates :password, presence: { message: "%{attribute} needs to have a value" }
+  
```

Comment:
Also you removed this important validation, good thing i found it in the merge conflicts and add it back